docs/getting-started/golden-path.md

```diff
@@ -0,0 +1,32 @@
+# The Golden Path
+
+Welcome to the world of Decred! We know that getting started with any new cryptocurrency
+can be difficult and we want to make the journey as easy as possible. Decred is fully
```

Comment:
'fully documented' seems a little too strong.  If it was fully documented we wouldn't have any open issues or PRs on the docs site :)

Maybe 'well documented' would be better.